firmware/user/user_main.c

```diff
@@ -90,8 +107,12 @@ static void ICACHE_FLASH_ATTR procTask(os_event_t* events);
 static void ICACHE_FLASH_ATTR updateDisplay(void* arg);
 static void ICACHE_FLASH_ATTR pollAccel(void* arg);
 
+#ifndef USE_2019_SWADGE
 static void ICACHE_FLASH_ATTR drawChangeMenuBar(void);
-
+#endif
```

Comment:
This could be an `#else`

Reply:
Im easy. BUT at this point USE_2019_SWADGE does not work. You mentioned at one point that it might be better to starr with 2019 and new modes, rather that mod the current to be backwards compatible. Maybe!